build/build.js

```diff
@@ -198,6 +211,19 @@ exports.test = function(complete, fail) {
 	console.log('Running tests...');
 
 	var server = new karma.Server(testConfig, function(exitCode) {
+
+		// Work around https://github.com/karma-runner/karma-slimerjs-launcher/issues/1
+		if (autoSlimer) {	// Only if we were able to run slimerjs and killall
+			var slimerjsPids = require('child_process').execSync('ps -Af | grep slimerjs | grep xulrunner | awk \'{print $2}\'').toString();
```

Comment:
I was kinda expecting that the

```js
   var child = require('child_process').execFileSync(   require('slimerjs').path  );
```

would throw an error in win32 and skip PhantomJS together.